appstore/appstore/settings/base.py

```diff
@@ -67,6 +81,24 @@
     'middleware.session_idle_timeout.SessionIdleTimeout'
 ]
 
+if DEBUG=="True" and DEV_PHASE in ("local", "stub", "dev"):
```

Comment:
@cnbennett3 would this work given the boolean change you did with DEBUG to make it work earlier?

Reply:
I'll test again with all the recent merges and make sure this still works/update for the change.